### PR TITLE
DV permission component stories

### PR DESCRIPTION
**Pull request by mdrillin**
changes related to TEIIDTOOLS-979

- Adds stories for a few of the permission components
- other minor DV story updates
- a few i18n additions
- minor layout adjustments

### CODE REVIEW

File: app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json [hunk 1]
```diff
@@ -107,6 +107,8 @@
   "permissionSelectAll": "Select all ({{allListLength}} items)",
   "permissionSet": "Set permissions",
   "permissionSeletedViews": "Grant or Revoke permissions for the selected Views: ",
+  "permissionShowLess": "Show less",
```
Comment (indraraj) [inline app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json:110]: these can be moved to shared and can remove permission from the key.

**Comment by indraraj** [comment]
not sure why we removed withNotes

**Comment by mdrillin** [comment]
> not sure why we removed withNotes

They were not showing up in storybook.  They were not removed, just changed the implementation which needed updating.

File: app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json (hunk 1 above)
Comment (mdrillin) [reply app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json:110]: agree - there are probably others that could be in shared too

**Pull request merged**